### PR TITLE
New package: taskd-1.1.0

### PR DESCRIPTION
**Pull request by stpx**

### CODE REVIEW

**Comment by pullmoll** [comment]
For musl libc you could add [`srcpkgs/taskd/patches/musl-limits_h.patch`](http://sprunge.us/gNZF).

File: srcpkgs/taskd/files/taskd/run [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+TASKDDATA=${TASKDDATA:-/var/lib/taskd}
```
Comment (xtraeme) [inline srcpkgs/taskd/files/taskd/run]: You can simplify this default assignments like this:

```
: ${TASKDDATA:-/var/lib/taskd}
...
```

**Comment by xtraeme** [comment]
Looks ok, thanks :+1:

**Comment by pullmoll** [comment]
I'm a bit late, as most of the times :+1:

**Pull request merged**